src/common/rule-based-view-model-provider.ts

```diff
@@ -9,20 +9,21 @@ import {
     CardRuleResult,
     CardRuleResultsByStatus,
     CardRuleResultStatus,
+    CardsViewModel,
 } from './types/store-data/card-view-model';
 import { UnifiedResult, UnifiedRule } from './types/store-data/unified-data-interface';
 
-export type GetUnifiedRuleResultsDelegate = (
+export type GetCardViewData = (
     rules: UnifiedRule[],
     results: UnifiedResult[],
     cardSelectionViewData: CardSelectionViewData,
-) => CardRuleResultsByStatus;
+) => CardsViewModel;
 
-export const getUnifiedRuleResults: GetUnifiedRuleResultsDelegate = (
+export const getUnifiedRuleResults: GetCardViewData = (
```

Comment:
Should this method be renamed to getCardViewData or something similar?

Reply:
Updated